caluma/caluma_core/urls.py

```diff
@@ -1,8 +1,17 @@
 from django.conf import settings
-from django.conf.urls import url
+from django.urls import re_path
 
+from caluma.caluma_core import views
 from caluma.caluma_user.views import AuthenticationGraphQLView
 
 urlpatterns = [
-    url(r"", AuthenticationGraphQLView.as_view(graphiql=settings.DEBUG), name="graphql")
+    re_path(
+        "graphql/?",
+        AuthenticationGraphQLView.as_view(graphiql=settings.DEBUG),
+        name="graphql",
+    ),
 ]
+
+# if enabled, include healthz endpoint path
+if settings.ENABLE_HEALTHZ_ENDPOINT:
+    urlpatterns.append(re_path("healthz/?", views.health_check_status, name="healthz"))
```

Comment:
Would be nice to have a small test that asserts a 404 if `ENABLE_HEALTHZ_ENDPOINT = False`.

Reply:
Added a test, it has to reload the URL confs though. Is there a way to provide different values for environment variables to different tests, so that the URL confs are loaded with the correct value for that test?

Reply:
Just had the idea that you could make the decision in the view instead of trying to reload/rebuild the URL conf

Return a HTTP 404 response if `settings.ENABLE_HEALTHZ_ENDPOINT` is `False`, and otherwise continue executing the checks